src/interfaces/SpokePool.ts

```diff
@@ -16,14 +16,13 @@ export interface DepositCommon {
   speedUpSignature?: string; // appended after initialization, if deposit was speedup (not part of Deposit event).
   updatedRecipient?: string;
   updatedMessage?: string;
-  realizedLpFeePct?: BigNumber; // appended after initialization (not part of Deposit event).
+  realizedLpFeePct: BigNumber; // appended after initialization (not part of Deposit event).
```

Comment:
OOC does the comment line up with the code? If we're appending this post-initialization then it should be `?:`?

Reply:
It's still correct; the comment refers to the fact that `realizedLpFeePct` is not emitted as part of the `FundsDeposited` event, so the `SpokePoolClient` computes and appends it to the event when it's being ingested.